src/lib/pipes/normalise-schedule-pipe.js

```diff
@@ -21,6 +21,7 @@ class NormaliseSchedulePipe extends Pipe {
             this.doCleanup();
             // There is a schedule, we need to parse it out into events.
             let {eventSchedule, ...parent} = this.rawData;
+            eventSchedule = Array.isArray(eventSchedule) ? eventSchedule : [eventSchedule];
```

Comment:
Just flagging that eventSchedule should always be an array, so this should come through as a validation error. Do you know which feed did not use an array?

Reply:
It doesn't look like this failure is properly handled and logged as an error, and the console output doesn't seem to show the publisher in most cases - just a null value. This might need a little more investigation.